var/spack/repos/builtin/packages/bazel/patch_for_fcc2.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/main/java/com/google/devtools/build/lib/bazel/rules/BazelRuleClassProvider.java
++++ b/src/main/java/com/google/devtools/build/lib/bazel/rules/BazelRuleClassProvider.java
+@@ -185,7 +185,7 @@ public class BazelRuleClassProvider {
+ 
+         Map<String, String> spackEnv = System.getenv();
+         for (String envName : spackEnv.keySet()) {
+-          if (envName.startsWith("SPACK_")) {
++          if ((envName.startsWith("SPACK_")) || (envName.equals("fcc_ENV")) || (envName.equals("FCC_ENV"))) {
```

Comment:
I think you'll need to do something similar for older versions as well.

Reply:
I tried to fix something similar for 0.24.1 or less as well. 
However, the following error occurred when I tried build bazel 0.24.1 on Fujitsu Compiler.

```
     386      /opt/FJSVxtclanga/tcsds-1.2.25/bin/fcc -U_FORTIFY_SOURCE -fstack-
            protector -Wall -Wthread-safety -Wself-assign -fcolor-diagnostics -
            fno-omit-frame-pointer -g0 -O2 '-D_FORTIFY_SOURCE=1' -DNDEBUG -ffun
            ction-sections -fdata-sections -MD -MF bazel-out/aarch64-opt/bin/th
            ird_party/zlib/_objs/zlib/adler32.d '-frandom-seed=bazel-out/aarch6
            4-opt/bin/third_party/zlib/_objs/zlib/adler32.o' -iquote . -iquote
            bazel-out/aarch64-opt/genfiles -iquote bazel-out/aarch64-opt/bin -i
            system third_party/zlib -isystem bazel-out/aarch64-opt/genfiles/thi
            rd_party/zlib -isystem bazel-out/aarch64-opt/bin/third_party/zlib -
            w '-Dverbose=-1' -no-canonical-prefixes -Wno-builtin-macro-redefine
            d '-D__DATE__="redacted"' '-D__TIMESTAMP__="redacted"' '-D__TIME__=
            "redacted"' -c third_party/zlib/adler32.c -o bazel-out/aarch64-opt/
            bin/third_party/zlib/_objs/zlib/adler32.o)
     387    Execution platform: @bazel_tools//platforms:host_platform
     388    [2 / 7] checking cached actions
     389
  >> 390    clang: error: unknown argument: '-fstack-protector'
     391    [2 / 7] checking cached actions
```

Therefore, I will set conflicts method as follows .What do you think of this idea ?

```
    conflicts('%fj', when='@:0.24.1',
          msg='Fujitsu Compiler can not build 0.24.1 or less, '
              'please use a newer release.')
```